SU-SWS/stanford_basic
# D8CORE-4490: adding secondary skip links to publications

# READY FOR REVIEW

# Summary
- Adding the skip link to the Publications pages.
- Changed the `a` on the secondary skip to link to a `div` to match what we did for the main skip.
- Added the skip to main as requested in D8CORE-4508 since had the other work in flight.

# Review By (Date)
- 7/5

# Criticality
- Med

# Urgency
- Med

# Review Tasks

## Setup tasks and/or behavior to test

1. Check out this branch
2. Rebuild Cache and import config `drush cr ; drush ci`
3. Create publications to test
4. Create a secondary menu
4. Navigate to pages with secondary menu
4. Verify you can use the skip to links

### Site Configuration Sync

- Is there a config:export in this PR that changes the config sync directory?

## Front End Validation
- [x] Automated accessibility: Scans performed? Or, https://stanfordits.atlassian.net/browse/D8CORE-4506

## Backend / Functional Validation
### Code

### Code security

## General

# Affected Projects or Products
- All sites using the SBT

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/D8CORE-4490
- https://stanfordits.atlassian.net/browse/D8CORE-4508 

# Resources
- [AMP Tool](https://stanford.levelaccess.net/index.php)
- [Accessibility Manual Test Script](https://docs.google.com/document/d/1ZXJ9RIUNXsS674ow9j3qJ2g1OAkCjmqMXl0Gs8XHEPQ/edit?usp=sharing)
- [HTML Validator](https://validator.w3.org/)
- [Browserstack](https://live.browserstack.com/dashboard) and link to [Browserstack Credentials](https://asconfluence.stanford.edu/confluence/display/SWS/External+Account+Credentials)
